src/modules/packages/packages.schema.yaml

```diff
@@ -44,8 +44,10 @@ properties:
                 #       need their own little schema.
                 install: { type: array }
                 remove: { type: array }
+                remove_deps: { type: array }
```

Comment:
`remove_deps` makes it sounds like it removes the dependencies only.  I think this name should be clearer.

Reply:
True. Atm I have these proposals in mind:
* try_remove_with_deps
* try_remove_w_deps
* try_remove_all

Ofc same for "remove_*".